tools/lldb-moduleimport-test/lldb-moduleimport-test.cpp

```diff
@@ -62,6 +62,33 @@ static void printValidationInfo(llvm::StringRef data) {
   }
 }
 
+static void resolveTypeFromMangledNameList(
+    swift::ASTContext &Ctx, llvm::SmallVector<std::string, 8> &MangledNames) {
+  std::string Error;
+  for (auto &Mangled : MangledNames) {
+    swift::Type ResolvedType =
+        swift::ide::getTypeFromMangledSymbolname(Ctx, Mangled, Error);
+    if (!ResolvedType) {
+      llvm::errs() << "Can't resolve type of " << Mangled << "\n";
+    } else {
+      ResolvedType->print(llvm::errs());
+      llvm::errs() << "\n";
+    }
+  }
+}
+
+static void
+collectMangledNames(std::string &FilePath,
+                    llvm::SmallVector<std::string, 8> &MangledNames) {
```

Comment:
…and SmallVectorImpl here (don't need to hardcode the size). Also, `const` on `FilePath`.